ntbs-service/appsettings.Production.json

```diff
@@ -7,6 +7,10 @@
     }
   },
   "ConnectionStrings": {
-    "ntbsContext": "<REPLACEME:ntbsContext>"
+    "ntbsContext": "<REPLACEME:ntbsContext>",
+    "auditContext": "<REPLACEME:auditContext>"
+  },
+  "AppConfig" : {
+    "AuditingEnabled" : false
```

Comment:
Should this be `true`? I think we want to have it turned on by default..

Reply:
Do we already want to start auditing then? Or shall we wait until we've figured out how we want to present the data, as that might affect which models we include etc.

Reply:
you are very right - i've jumped the gun to already collecting it. doh!